db/migration/V019__create_programs.sql

```diff
@@ -1,21 +1,22 @@
 CREATE TABLE curriculums (
   id BIGINT NOT NULL AUTO_INCREMENT,
+  title VARCHAR(255) NOT NULL,
   principal_id BIGINT NOT NULL,
   created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
   updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
-  title VARCHAR(255) NOT NULL,
   INDEX curriculums_principal_id (principal_id),
   FOREIGN KEY (principal_id) REFERENCES principals(id),
   PRIMARY KEY (id)
 );
 
 CREATE TABLE programs (
   id BIGINT NOT NULL AUTO_INCREMENT,
-  principal_id BIGINT NOT NULL,
```

Comment:
Good job for reorganizing the columns by moving the more important columns to the top.